handler/actions/src/main/java/io/knotx/fragments/handler/action/http/WebClientCache.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (C) 2019 Knot.x Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.knotx.fragments.handler.action.http;
+
+import com.google.common.cache.Cache;
+import com.google.common.cache.CacheBuilder;
+import io.vertx.core.Vertx;
+import io.vertx.core.json.JsonObject;
+import io.vertx.ext.web.client.WebClientOptions;
+import io.vertx.reactivex.ext.web.client.WebClient;
+
+class WebClientCache {
+
+  private static final long WEB_CLIENT_CACHE_SIZE = 200;
```

Comment:
The config value would be quite "out of scope" for the ActionNodeFactory. The `cache.maxSize` value in here is passed, but not associated with the actual ActionFactory that uses it. The `ServiceLoader.load(ActionFactory.class).iterator()` method is used for lazy instantiation of ActionFactories, but it is not clear to me, how could configuration be provided in here.
We could change the ActionFactory configuration/instantiation mechanism, but it would change the API significantly and I think it would be out of scope for this PR.

Reply:
Described under issue #72